language/diem-framework/modules/Genesis.move

```diff
@@ -89,6 +93,86 @@ module Genesis {
         DiemTimestamp::set_time_has_started(dr_account);
     }
 
+    fun create_initialize_owners_operators(
```

Comment:
I'd love a comment explaining this.  I had to get well into the function to understand that "owners" is a vector of validator owners.   I'll have to finish reading the function then look for where it's called to understand where the args come from.

Reply:
Good idea, I had to do a lot of investigating to understand the old Rust code, will try to encode that knowledge in comments.